### DISCUSSION TITLE
Add today's date?

### DISCUSSION THREAD

**Discussion by ali-gangji** [sql-bi/Bravo#442] [Q&A] [answered]
Silly question: Is there a way to add a column for today's date? Thanks in advance.

**Comment by albertospelta** [comment]
The current version does not allow adding a new column to the calendar table, this is an operation that can still be done manually by editing the table expression.

``` sql
Date = ADDCOLUMNS(
    DateAutoTemplate,
    "Today", TODAY()
)
```

We are working on a system to provide customized templates for date tables and the plan is to release such a feature for v1.0.

**Comment by marcosqlbi** [comment, marked as answer]
@ali-gangji You can now create custom templates: we will complete the full documentation for v1.0, in the meantime, you can already try it and use this documentation: https://docs.sqlbi.com/bravo/features/manage-dates/customize-date-template
